app/templates/search.hbs

```diff
@@ -2,14 +2,10 @@
   <div class="alert alert-danger">{{errorMessage}}</div>
 {{/if}}
 
-<form class="form-inline" {{action "search" on="submit"}}>
-  <div class="form-group">
-    {{focus-input value=query class="form-control" required=true}}
-  </div>
-  <button type="submit" class="btn btn-default" disabled={{isSearching}}>
-    Search <span class="glyphicon glyphicon-refresh {{if isSearching '' 'hidden'}}"></span>
-  </button>
-</form>
+<div class="search-bar">
+  {{focus-input value=query placeholder='Search...' class="form-control" keyPressAction="search"}}
```

Comment:
When you leave this input inside the form and remove submit button, it will still submit when pressing enter, so no need for https://github.com/cowbell/mopster/pull/4/files#diff-215c693c56041f59d7446f52e68aee55R9